### ISSUE TITLE
programming style

### ISSUE THREAD

**Issue by qzhu2017** [davidz873/CMSUNLV#2]
@davidz873 

When you define the function, please write something in the function add describe the purpose, input and output.
These information could be extremely useful for code review.
```python
def smear(data, sigma):
    """
    Apply Gaussian smearing to spectrum y value.
    Args:
        data: a 2 dimensional data
        sigma: Std dev for Gaussian smear function
    Returns:
        a new spectrum with updated y value
    """
```

**Comment by qzhu2017** [comment]
@davidz873 
Something maybe tedious. It is good to follow the pep8 style with your code.

A simple way is to use the pycodestyle lib.
https://pypi.org/project/pycodestyle/

**Comment by qzhu2017** [comment]
@davidz873  has been fixed in cc52e4843fcc61e8d1e34f5758ba7f3b74334e11
